Tests/StreamChatTestTools/Mocks/MockEndpointResponses/Member.json

```
@@ -5,7 +5,7 @@
   "shadow_banned" : true,
   "ban_expires" : "2021-03-08T15:42:31.355923Z",
   "user_id" : "broken-waterfall-5",
-  "role" : "owner",
+  "channel_role" : "custom_role",
```

nit: what if we change only the JSON key here? This would keep the existed test untouched and make sure it keeps working. 

Then the new JSON (`Member+CustomRole`) can contain the custom role + the new test would verify custom roles are also successfully decoded.
```suggestion
  "channel_role" : "owner",
```

---

Ah! I thought I changed this back, yep this makes sense - let me push the change